ui/src/app/edge/live/chpsoc/chpsoc.component.html

```diff
@@ -0,0 +1,46 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="edge.currentData | async as currentData">
+        <ion-card>
+            <ion-item lines="full" color="light">
+                <ion-avatar slot="start">
+                    <ion-icon name="flame" style="font-size: 40px" font></ion-icon>
+                </ion-avatar>
+                <ion-label translate>CHP</ion-label>
```

Comment:
This `translate` (and others below) won't work. The text inside the tag needs to be a 'path' to a translated string. We are using https://github.com/ngx-translate/core for that. Translations are e.g. here https://github.com/OpenEMS/openems/blob/develop/ui/src/app/shared/translate/de.ts